chall02/rpehkone.c

```diff
@@ -0,0 +1,24 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+char	*hv_rgb2hex(int r, int g, int b)
+{
+	char *res;
+	char hex[3];
+
+	if (r < 0 || g < 0 || b < 0 || r > 255 || g > 255 || b > 255)
```

Comment:
nice you checked the user input.

a small function `color_is_valid(int c)` would make it cleaner